aiogram/dispatcher/middlewares.py

```diff
@@ -118,8 +118,13 @@ async def post_process(self, obj, data, *args):
         pass
 
     async def trigger(self, action, args):
-        if self.skip_patterns is not None and any(item in action for item in self.skip_patterns):
-            return False
+        if self.skip_patterns:
+            skip_actions = (
```

Comment:
What do you think about moving this variable to __init_subclass__ or property with caching to prevent to re-generate this list of method names for each event?